### ISSUE TITLE
The notify_summary thread does not handle exceptions through the normal means

### ISSUE THREAD

**Issue by laxdog** [BitBotFactory/poloniexlendingbot#261]
Exceptions on this thread don't get passed to lendingbot.py, so aren't handled gracefully

```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 811, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 764, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/sched.py", line 117, in run
    action(*argument)
  File "/home/dog/poloniexlendingbot_market_anal/modules/Lending.py", line 97, in notify_summary
    log.notify(Data.stringify_total_lended(*Data.get_total_lended()), notify_conf)
  File "/home/dog/poloniexlendingbot_market_anal/modules/Data.py", line 45, in get_total_lended
    crypto_lended = api.return_active_loans()
  File "/home/dog/poloniexlendingbot_market_anal/modules/Poloniex.py", line 159, in return_active_loans
    return self.api_query('returnActiveLoans')
  File "/home/dog/poloniexlendingbot_market_anal/modules/Poloniex.py", line 104, in api_query
    ret = urllib2.urlopen(urllib2.Request('https://poloniex.com/tradingApi', post_data, headers))
  File "/usr/lib64/python2.7/urllib2.py", line 154, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib64/python2.7/urllib2.py", line 437, in open
    response = meth(req, response)
  File "/usr/lib64/python2.7/urllib2.py", line 550, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib64/python2.7/urllib2.py", line 475, in error
    return self._call_chain(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 409, in _call_chain
    result = func(*args)
  File "/usr/lib64/python2.7/urllib2.py", line 558, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 422:
```

Not the end of the world, but should be fixed.

**Comment by kryztoval** [comment]
True, I disabled the Raise in that code because it would send me the code to Telegram both... complaining the ID was incorrect but showing me the error in chat. So odd.

**Comment by kryogenic** [comment]
This should be fixed now #281

**Comment by laxdog** [comment]
Thanks @kryogenic
